src/ingest/raw_val.rs

```diff
@@ -17,6 +18,15 @@ impl RawVal {
             RawVal::Null => BasicType::Null,
         }
     }
+
+    pub fn heap_size_of_children(&self) -> usize {
+        match *self {
+            RawVal::Int(_) => mem::size_of::<i64>(),
```

Comment:
This is an underestimate because the `RawVal` will always be large enough to hold the largest value + a discriminant value (which will total 32 bytes or so).
Maybe just return heap size of any strings as is and 0 for other variants, and then have another `self.data.capacity() * size_of::<RawVal>` term in `MixedCol::heap_size_of_children` to account for empty slots as for maximum accuracy.